internal/config/config.go

```diff
@@ -34,25 +34,31 @@ type Default struct {
 
 	// TZ represent system time location .
 	TZ string `json:"time_zone" yaml:"time_zone"`
+
+	// Log represent log configuration.
+	Log Log `json:"log" yaml:"log"`
```

Comment:
Log type should be pointer for checking Bind

Reply:
sorry.! I wil fix soon